transpire/helpers/helm.py

```diff
@@ -95,6 +95,35 @@ def build_chart(
     if shutil.which("helm") is None:
         raise RuntimeError("You must install Helm to use this script.")
 
+    # make sure the current user has a place for us to cache stuff
+    home_cache = os.environ.get('XDG_CACHE_HOME', None)
```

Comment:
I think the way we want this to work is...

1. If `TRANSPIRE_CACHE_DIR` is set, use `$TRANSPIRE_CACHE_DIR/charts` as the cache directory.
2. Failing that, if `XDG_CACHE_HOME` is set, use `$XDG_CACHE_HOME/transpire/charts` as the cache directory.
3. Failing that, if `HOME` is set, use `$HOME/.cache/transpire/charts` as the cache directory.
4. Failing that, use `/tmp/transpire/charts` as the cache directory.

Having this fail seems like an annoyance when before it worked in all cases. Also, this way we can support running transpire in CI with a persistent cache!

Reply:
That makes a lot of sense. I figured that we'd want to pick another cache location for transpire at some point without affecting `$XDG_CACHE_HOME`, but I wasn't sure how to go about it.